### DISCUSSION TITLE
Implement Percolate Mode

### DISCUSSION THREAD

**Discussion by AbstractiveNord** [meilisearch/product#673] [Feedback & Feature Proposal]
Ask for implement Percolate Mode, named also Reverse Search.
Great if possible to connect some MQs for that type of search.

**Comment by macraig** [comment]
Hi @AbstractiveNord , thanks for taking the time to leave your feedback. It has inspired a potential new feature, which we've added to our [public roadmap](https://roadmap.meilisearch.com/c/195-percolator-query-reverse-search). It would be great if you upvote the feature, as your input helps us prioritize our future development efforts.

Thanks again for being a part of our product's growth!

**Reply by techpeace** [reply]
Howdy! My team was just discussing this Elasticsearch feature as we're evaluating search platform offerings for our agent product (including Meilisearch). Ran across your post, but I couldn't find this listed on the roadmap. Just wondering where this ended up landing.

Thank you!
